README.md

```
@@ -8,19 +8,105 @@ Azure deployment (Docker Compose): http://4.223.70.80/
+This starts
+- the web client: http://localhost:8080
+- the Spring API: http://localhost:8081
+- the GenAI services on the internal network
+- swaggerUI documentation for public API: http://localhost:8083/openapi/openapi.yaml
+- swaggerUI documentation for internal API: http://localhost:8083/openapi/openapi-internal.yaml
+
+## Architecture
+
+### System component diagram
+
+![](doc/architecture.svg)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)